pemFioi/beaver-task-2.0.js

```diff
@@ -214,10 +214,14 @@ function initWrapper(initSubTask, levels, defaultLevel, reloadWithCallbacks) {
             if(window.displayHelper) {
                displayHelper.displayLevel(state.level);
             }
+            var answer = mainTask.answer;
             mainTask.loadLevel(state.level, null, views);
             if(levels) {
                displayHelper.setupLevels(defaultLevel, reloadWithCallbacks, levels);
             }
+            if (answer) {
+               mainTask.reloadAnswerObject(answer)
```

Comment:
These modifications don't have anything to do in a common file used by lots of tasks, many which aren't using quickAlgo. It's really best to avoid modifying files which are outside the scope of your project, when you don't know whether it could break completely unrelated tasks.

Reply:
Unfortunately, I didn't found any better place to do this, maybe you know where it could be put? And normally nothing should happens if answer is null which is the case in 99% of cases.

The only case it could break is if mainTask is null I don't know if it happens in a weird subject.

Reply:
It should be put after loading the data from the remote task. Where are you setting mainTask.answer ? You should reload the answer from there instead.

> nothing should happens if answer is null which is the case in 99% of cases

Are you sure of that? I'm pretty sure you haven't checked all the Bebras tasks :)